docs/website/src/page/articles-layout.less

```diff
@@ -22,4 +23,15 @@
     color: @nav-accent;
     margin-bottom: 40px;
   }
+
+  &-content {
+    margin-bottom:5px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: -webkit-box;
+    -webkit-line-clamp: 3; /* number of lines to show */
+    line-clamp: 3; 
```

Comment:
as I can see, in every browser this property is used as `-webkit-line-clamp`, mb can we remove this line?